tutorials/bigtable-stackdriver/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Using matplotlib to visualize Stackdriver metrics for Bigtable
+description: Learn about using matplotlib to plot Stackdriver metrics for Bigtable
+author: waprin
+tags: Stackdriver, Bigtable, matplotlib
+date_published: 2017-06-13
+---
+
+[Google Stackdriver Monitoring](https://cloud.google.com/monitoring/docs/) is
```

Comment:
For consistency, the link should point to the product page (https://cloud.google.com/monitoring/).